components/SearchPageJumbotron.js

```diff
@@ -109,11 +111,18 @@ function SearchPageJumbotron() {
         <form onSubmit={onSearch} className={classes.form}>
           <h2 className={classes.search}>{t`Searching`}</h2>
           <Box flex={1} className={classes.inputArea}>
-            <textarea
+            <TextareaAutosize
               ref={textareaRef}
               name="search"
               className={classes.input}
-              rows={1}
+              minRows={1}
+              onKeyDown={e => {
+                // enter to trigger search, so disable default line breaking
+                if (e.key == 'Enter') {
```

Comment:
I am thinking if we should allow line break when shift-enter is pressed. What do you think?

Reply:
I haven't looked into how search works, but given that the use case is searching, I personally think that users will input space separated texts instead of line separated texts.

Reply:
Agree. it's just that when user are provided with a multiline text area that submits when hitting enter, they may expect a soft-return mechanism (usually shift+enter) so that they can enter new lines. An example would be message textarea of a chat window.

But this is not mandatory for Cofacts' search box; I would expect that most user would copy and paste suspicous text from elsewhere and directly hit enter.